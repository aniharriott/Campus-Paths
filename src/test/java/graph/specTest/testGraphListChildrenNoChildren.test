# Test Graph listChildren no children file.
# Tests that no children will be listed is
# there are no children nodes of the specified
# node.

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Print the children from n1
ListChildren graph1 n1