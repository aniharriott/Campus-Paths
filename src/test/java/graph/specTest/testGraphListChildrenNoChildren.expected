# Test Graph listChildren no children file.
# Tests that no children will be listed is
# there are no children nodes of the specified
# node.

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Print the children from n1
the children of n1 in graph1 are: